include/sound/sof/ipc4/header.h

```diff
@@ -185,6 +185,9 @@ enum sof_ipc4_pipeline_state {
 #define SOF_IPC4_GLB_PIPE_STATE_MASK		GENMASK(15, 0)
 #define SOF_IPC4_GLB_PIPE_STATE(x)		((x) << SOF_IPC4_GLB_PIPE_STATE_SHIFT)
 
+#define SOF_IPC4_GLB_PIPE_MULTI			BIT(0)
+#define SOF_IPC4_GLB_PIPE_MULTI_SYNC_STOP	BIT(1)
```

Comment:
SOF_IPC4_GLB_PIPE_MULTI_SYNC_STOP is for what?

so, this is a batch pipeline trigger feature?

Reply:
it is also needed for multiple pipeline trigger but when & for what is an open for now. But I've added it as it is part of the multi-pipeline trigger IPC bits

Reply:
Erm, what is the open? That statement doesn't give me a warm and fuzzy feeling we should merge this. This looks like we're going to have multiple revisions if there are open conceptual issues on how this multi pipe stuff works.

Reply:
i dont think there's anyting conceptual missing here @plbossart. SYNC_STOP is a feature with multi-pipeline trigger that we just dont need right now

Reply:
I find it confusing to add references to a TBD feature that we don't need. Let's add it if and when we have a need and actually use it.